src/proposals.cairo

```diff
@@ -554,5 +579,15 @@ mod proposals {
             self.custom_proposal_type.write(idx, config);
             idx
         }
+
+        fn set_default_proposal_params(ref self: ComponentState<TContractState>, quorum: u32, proposal_voting_seconds: u32) {
```

Comment:
Looks good for proposal parameter configurations. Only potential suggestion is maybe emitting an event for changes in proposal parameters?

Reply:
No strong opinion on that, but don't see a reason to consume that event, so not adding